hera_cal/utils.py

```diff
@@ -1049,16 +1048,16 @@ def red_average(data, reds=None, bl_tol=1.0, wgt_by_int=False, inplace=False,
             If None, will calculate these from the metadata
         bl_tol : float
             Baseline redundancy tolerance in meters. Only used if reds is None.
-        wgt_by_int : bool
-            Weight average by total integration time (nsamples * integration_time * ~flags)
-            Otherwise weighting is just uniform (1 * ~flags)
         inplace : bool
             Perform average and downselect inplace, otherwise returns a deepcopy.
             The first baseline in each reds sublist is kept.
+        wgts : DataContainer
+            Manual weights to use in redundant average. This supercedes flags and nsamples
+            if provided, and will also be used if input data is a UVData or a subclass of it.
         flags : DataContainer
             if data is a DataContainer, these are its flags
```

Comment:
I'd mention what happens if data is a DataContainer but flags and/or nsamples is not provided and also wgts are not provided